introduction.tex

```diff
@@ -1,2 +1,40 @@
+SymPy is an full featured computer algebra system (CAS) written in the Python
+programming language. It is open source, licensed under the extremely
+permissive BSD license, which allows anyone to reuse SymPy or its source code,
+even for commercial purposes. SymPy was started by Ond\v{r}ej \v{C}ert\'{\i}k
+in 2005, and it has since grown into a large open source project, with over
+500 contributors.
+% citation?
 
+SymPy is written entirely in the Python programming language,
+% cite Python?
+which is also the language used to interact with it, both programmatically and
+interactively. Python is a popular dynamically typed programming language that
+has a focus on ease of use and readability. Unlike many CASs, SymPy does not
+invent its own programming language. We rather take the approach that Python
+is already a well-designed programming language, built by expert language
+designers, something which the majority of SymPy developers are not. Python is
+also a very popular language for scientific computing and data science, with a
+wide range of useful libraries.
+% Cite numpy, scipy, pandas
+% We could also cite
+% https://stackoverflow.com/research/developer-survey-2016#most-popular-technologies-per-occupation
 
+SymPy is designed with a strong focus that it be usable as a library. This
+means that extensibility is important in its API design. This is also one of
+the reasons SymPy makes no attempt to extend the Python language itself. The
```

Comment:
I think the dynamic portions don't need to be discussed here.  Maybe mentioned as part of the tools sections.